src/Rokt-Kit.js

```diff
@@ -55,8 +68,15 @@ var constructor = function () {
         var accountId = settings.accountId;
         self.userAttributes = filteredUserAttributes;
         self.onboardingExpProvider = settings.onboardingExpProvider;
+        self.vNextExtensions = extractvNextExtensions(settings.vNextExtensions);
```

Comment:
if this is only being used once, there isn't a reason for it to be on the class.  it can just be declared as a variable nd passed into line 73, right?